minecraft_servers/blockleben/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+  "server_name": "blockleben",
+  "nice_name": "BlockLeben",
+  "direct_ip": "blockleben.de",
+  "server_wildcards": [
+    "blockleben.net"
+  ],
+  "supported_languages": [
+      "de"
+  ],
+  "social": {
+    "web": "https://blockleben.de",
```

Comment:
Website is file upload?

Reply:
Our Website is in maintenance
must there be a website?

Reply:
If web is still in maintenance, please remove it; you can add it once there's a website

Reply:
i cant create a pull request